databricks/koalas/series.py

```diff
@@ -3804,6 +3804,86 @@ def where(self, cond, other=np.nan):
 
         return result
 
+    def mask(self, cond, other=np.nan):
```

Comment:
Is this the same as `where` except for the opposite cond?
If so, how about just reusing it like `return self.where(~cond, other)`?

Reply:
thanks for the comment!! what a gorgeous way it is. 👍

Reply:
i just fixed it !
ah, i should have taken a look at functionalities once more before implement something.